settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "workbench.settings.editor": "json",
+    "python.defaultInterpreterPath": "~/miniconda3/envs/mle-dev/bin/python",
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+    }
+}
```

Comment:
You can isort and black configurations as well, flake8 can be set as default linter

Reply:
Yes sir `isort`,`black` and `flake8` have already been updated in global (user) settings. I am attaching the screen shot of user settings for your review here.
![image](https://user-images.githubusercontent.com/98591251/175283508-c7f98515-b234-4308-8f6f-3806e73d60ed.png)

Reply:
Usually having `setting.json` at the repo level helps